### DISCUSSION TITLE
MouseWhのキーを動作させたい

### DISCUSSION THREAD

**Discussion by Nunu-tanu** [mentako-ya/mtk#22] [Q&A] [answered]
左にエンコーダー、右にトラックボールの環境で
mtk_mtk64e_PMW3389_AutoMouseLayer_via.0.22を使用しています。
remapを使用してエンコーダにマウスのスクロール機能を実装させたいのですが
MouseWh↓等のキーを配置しても動作しませんでした。（エンコーダ以外のボタンでも同様でした）

事前に設定等が必要なのでしょうか。

**Comment by mentako-ya** [comment, marked as answer]
[rules.mk](https://github.com/mentako-ya/mtk/blob/main/qmk_firmware/keyboards/mtk/mtk64e/rules.mk)に
MOUSEKEY_ENABLE = yes
の指定を追加してビルドしてください。
ただし、ファームウウェア容量の上限を超えるため、代わりに何かの機能を無効化する必要があります。
